apps/web/src/lib/schema.ts

```diff
@@ -1,33 +1,51 @@
-import { z } from "zod";
+import { z } from 'zod';
 
 export const formSchema = z.object({
```

Comment:
this should be named more accurately

Reply:
wdym it should be renamed more accurately? It's a line importing zod